### PR TITLE
[Naitei][Gem toast rails]  Toast message

### PR DESCRIPTION
**Pull request by SuKay712**
## Related Tickets
https://edu-redmine.sun-asterisk.vn/issues/79545
## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)

https://github.com/user-attachments/assets/a73ae13c-60ed-4c80-a5e1-cbbe879f2612

### CODE REVIEW

**Comment by SuKay712** [comment]
ready

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -21,4 +21,23 @@ def format_currency amount
...
+      escaped_message = j(message)
+      flash_messages << <<~HTML
+        <script>
+          document.addEventListener("DOMContentLoaded", function() {
+            toastr.#{type}("#{escaped_message}");
+          });
+        </script>
+      HTML
+    end
+    flash_messages.join("\n").html_safe # rubocop:disable Rails/OutputSafety
```
Comment (ghost) [inline app/helpers/application_helper.rb]:
```suggestion
    flash_messages.join("\n")
```
em html_safe ở chỗ ngoài view nhé

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -14,11 +14,12 @@
     <%= stylesheet_link_tag "wishlist", "data-turbo-track": "reload" %>
     <%= stylesheet_link_tag "cart", "data-turbo-track": "reload" %>
     <%= stylesheet_link_tag "bill", "data-turbo-track": "reload" %>
+    <%= stylesheet_link_tag "https://cdnjs.cloudflare.com/ajax/libs/toastr.js/2.1.3/toastr.min.css" %>
     <%= javascript_importmap_tags %>
+    <%= custom_bootstrap_flash %>
```
Comment (ghost) [inline app/views/layouts/application.html.erb]:
```suggestion
    <%= custom_bootstrap_flash.html_safe %>
```

**Review by ghost — COMMENTED** [review]
cmt

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (SuKay712) [reply app/views/layouts/application.html.erb]: Dạ em sửa rồi ạ

**Comment by SuKay712** [comment]
ready

**Comment by SuKay712** [comment]
ready

**Review by ghost — APPROVED** [review]

**Pull request merged**